Cargo.toml

```diff
@@ -31,7 +31,7 @@ futures = "0.3.21"
 num_cpus = "1.13.1"
 serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
-tokio = { version = "1.15.0", features = ["full", "rt", "time", "macros", "test-util"] }
+tokio = { version = "1.26.0", features = ["full", "rt", "time", "macros", "test-util"] }
```

Comment:
Not sure why this change is here. This was already set. I wonder if GitHub's viewer is bugged?

Reply:
This one was for the dev-dependency.

Reply:
Ah